### ISSUE TITLE
Restrict forums per site

### ISSUE THREAD

**Issue by stianpr** [ellmetha/django-machina#36]
This is probably just a feature request or just a question to how one can restrict forums to a site.
We are currently using the [sites framework](https://docs.djangoproject.com/en/1.10/ref/contrib/sites/) in Django pretty heavily and it would be super awesome to restrict a created forum/category to a site. How easy do you think this is? I would imagine we'd have to filter by site id on the queryset and add another field on the forum model to specify a site. If this is something that you don't see as a new feature in this project, how easy can one modify the forum model?

**Comment by stianpr** [comment]
I should have read the great documentation before I posted this.  So I easily managed to extend the forum model with a site foreign key. But have you ever thought about having a site relation on the models?

**Comment by ellmetha** [comment]
> But have you ever thought about having a site relation on the models?

No. But your use case is interesting. I could plan to tie machina's forums with Django's sites in further releases (you could also submit a pull request if you have time to work on this!). Of course this would involve updating many views of *django-machina* in order to filter the forums by site ID.
